File: apps/api/api/health.js
// Simple health check endpoint - no database required
module.exports = (req, res) => {
  res.status(200).json({
    status: 'ok',
    timestamp: new Date().toISOString(),
    service: 'smart-dispatch-api',
    version: '1.0.0'
  });
};

File: apps/api/src/handlers/health.rs
use axum::Json;
use serde_json::json;

pub async fn health_check() -> Json<serde_json::Value> {
    Json(json!({
        "status": "ok",
        "service": "smart-dispatch-api",
        "version": "1.0.0"
    }))
}
